test/e2e/serial/tests/tolerations.go

```diff
@@ -784,10 +830,63 @@ func sriovToleration() corev1.Toleration {
 	}
 }
 
-func waitForMcpUpdate(cli client.Client, ctx context.Context, mcps []*machineconfigv1.MachineConfigPool) {
-	By("waiting for mcp to start updating")
-	waitForMcpsCondition(cli, ctx, mcps, machineconfigv1.MachineConfigPoolUpdating)
+func waitForMcpUpdate(cli client.Client, ctx context.Context, mcpsInfo []mcpInfo, updateType MCPUpdateType) {
```

Comment:
we can move the variadic part at the end, and use an actual variadic argument:
```
func waitForMcpUpdate(cli client.Client, ctx context.Context, updateType MCPUpdateType, mcpsInfo ...mcpInfo) {
```

so we can write
```
waitForMcpUpdate(cli, ctx, myUpdate, mcp1)
...
waitForMcpUpdate(cli, ctx, myUpdate, mcp1, mcp2, mcp3)
...
waitForMcpUpdate(cli, ctx, myUpdate, mcpList...)
```

Reply:
ditto for `waitForMcpsCondition`
nit: we should always UPPERCASE ACRONYMS, so it should be MCPs, but the codebase is already inconsistent